### DISCUSSION TITLE
How can I Configure the Domain to Separate Web Hosting on Vercel and Email Services on Cpanel?

### DISCUSSION THREAD

**Discussion by suskarkhy** [vercel/community#4486] [Help]
### Summary

I'm writing to seek your guidance and assistance regarding our domain configuration on Vercel. We currently have our domain connected to a server (using Cpanel), which handles our email services, and we'd like to deploy our website using Vercel. However, when I changed the A record to point to Vercel's server, I encountered issues with our email services. Specifically, we received the following error message: "SMTP error from remote mail server after MAIL FROM unable to verify sender." no one was able to send or receive any emails.

I suspect that changing the A record to Vercel may have disrupted our email services. Our primary goal is to maintain the email services on our current server while ensuring that our website is deployed on Vercel.

Could you please provide me with instructions on how to configure our domain so that it directs web traffic to Vercel while keeping our email services functioning on our existing server? I want to achieve this separation of services without any disruption to our email capabilities.

If there are specific DNS or configuration settings I should be aware of, please provide detailed instructions to ensure a smooth transition.

### Steps to Reproduce

1. Deployed the website on Vercel.
2. Accessed the domain's DNS settings in cPanel.
3. Modified the A record, changing it from the original server's IP address to Vercel's server IP address.

**Comment by Availeth** [comment]
Were you able to solve this problem? I'm currently having the same issue, and i haven't been able to find a solution. I tried following the step in this article "[https://remjx.com/how-to-setup-c-panel-email-on-vercel-or-any-other-dns-host]" but it didn't work. Please share if you fond a solution, Thanks.
